internal/buffer/search.go

```diff
@@ -6,6 +6,40 @@ import (
 	"github.com/zyedidia/micro/v2/internal/util"
 )
 
+func padRegexp(r *regexp.Regexp) (*regexp.Regexp, *regexp.Regexp, *regexp.Regexp, *regexp.Regexp) {
+	rPadStart := regexp.MustCompile(".(?:"+r.String()+")")
+	rPadEnd := regexp.MustCompile("(?:"+r.String()+").")
+	rPadBoth := regexp.MustCompile(".(?:"+r.String()+").")
+	return r, rPadStart, rPadEnd, rPadBoth
+}
+
+func findLineParams(b *Buffer, start, end Loc, i int, rPadded [4]*regexp.Regexp) ([]byte, int, *regexp.Regexp) {
+	l := b.LineBytes(i)
+	charpos := 0
+	ri := 0 // rPadNone
```

Comment:
The idiomatic way to define such C-enum-style numeric constants is:
```
const (
	rPadNone = iota
	rPadStart
	rPadEnd
	rPadBoth
)
```

Reply:
done